pkg/sql/parser/sql.y

```diff
@@ -4909,6 +4910,7 @@ drop_stmt:
 | drop_role_stmt     // EXTEND WITH HELP: DROP ROLE
 | drop_schedule_stmt // EXTEND WITH HELP: DROP SCHEDULES
 | drop_external_connection_stmt // EXTEND WITH HELP: DROP EXTERNAL CONNECTION
+| drop_tenant_stmt              // EXTEND WITH HELP: DROP TENANT
```

Comment:
Drive by nit: We might want to keep this out of the help until we decide if we're exposing this for customer use.